visualization/SensorVisualization/SensorVisualization.pde

```diff
@@ -0,0 +1,117 @@
+/*
+ * SensorVisualization
+ *
+ * (c) 2016 Brian Dolan, RoboSail
+ *
+ * NOTE: This program currently requires Processing version 1.5.1.  Later versions of Processing
+ * currently are not supported.
+ *
+ * This program displays a 3D visualization of a RoboSail sailboat's sensor values, including
+ * roll, pitch, yaw, (from the accelerometer) and wind direction (from the wind direction encoder).
+ * It reads the values through the serial port, and is made to work with the RoboSail Arduino
+ * programs.  It is intended to be used a tool to aid in writing and debugging code on the RoboSail
+ * Arduino processor.
+ *
+ * Angles of the rudder, sail, and wind sensor on the sailboat are expressed like this:
+ *
+ *            (overhead view)
+ *                   /                    ^ 90 degrees
+ *            ,,----/-------/             |
+ *      bow  <     +      + |  stern      +----> 0 degrees
+ *            ``------------+             |
+ *                                        v -90 degrees
+ */
+// TODO: better-looking boat
+//       support for Processing 3: textMode(SCREEN), water transparency, axes labelling
```

Comment:
I noticed that the waves don't accurately reflect the wind state or the boat wake. Pls add. KTHXBYE.